Card/kakita-yoshi.json

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "clan": "crane",
+        "cost": 5,
+        "deck_limit": 3,
+        "element": null,
+        "fate": null,
+        "glory": 3,
+        "honor": null,
+        "id": "kakita-yoshi",
+        "influence_cost": null,
+        "influence_pool": null,
+        "military": "2",
+        "military_bonus": null,
+        "name": "Kakita Yoshi",
+        "political": "6",
+        "political_bonus": null,
+        "role_restriction": null,
+        "side": "dynasty",
+        "strength": null,
+        "strength_bonus": null,
+        "text": "<b>Action:</b> While this character is participating in a conflict, discard the Imperial Favor – draw 3 cards. Reduce the cost of each event you play during this conflict by 2.",
+        "traits": [
+            "courtier",
+            "daimyō",
```

Comment:
@mplain traits must be lowercase ascii, like "id" -> "daimyo"

Reply:
Got it!